### ISSUE TITLE
Not compatible with apps that use a strict `or` helper

### ISSUE THREAD

**Issue by runspired** [miguelcobain/ember-leaflet#636]
Had a fun debugging session tonight 😭

My app defines an `or` helper that does not come from `ember-truth-helpers`. This `or` helper is strict vs loose (meaning that it casts the return value to a boolean rather than returning a falsey or truthy value).

Turns out that the component assignment is depending on the loose `or`

```hbs
{{#each this.componentsToYield as |c|}}
      {{ember-leaflet-assign-to
        components
        key=(or c.as c.name)
        value=(component c.name parent=this node=Node)
        onChange=this.mergeComponents}}
    {{/each}}
  ```

**Comment by miguelcobain** [comment]
I see. Perhaps we should include our own "loose" `or` helper.

**Comment by miguelcobain** [comment]
This fix was released on `5.0.0-beta.4`.
